contracts/transfer/circuits/src/dusk_contract.rs

```diff
@@ -0,0 +1,12 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+
+pub mod send_to_contract_transparent;
+pub mod send_to_contract_obfuscated;
+pub mod withdraw_from_obfuscated;
+pub mod withdraw_from_obfuscated_to_contract;
+pub mod execute;
```

Comment:
Instead of importing the modules publicly. Import them without the `pub` and do a `pub use` for the specific Circuit implementations. Will look cleaner.